adm/templates/plugins/django/{{cookiecutter.name}}/python2_virtualenv_sources/.gitignore

```diff
@@ -0,0 +1,2 @@
+requirements2.txt
+src
```

Comment:
please put a test like `{% if cookiecutter.type == "gunicorn2_sync" %}` before these lines
if we selected python3, the file will be empty (with this test) and deleted as the `python2_virtualenv_sources` directory

Reply:
done